dynamicly quantized lstm benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#30149 dynamicly quantized lstm benchmarking**

Differential Revision: [D18613005](https://our.internmc.facebook.com/intern/diff/D18613005)

Commit:
dynamicly quantized lstm benchmarking

Commit:
Update on "dynamicly quantized lstm benchmarking"


Differential Revision: [D18613005](https://our.internmc.facebook.com/intern/diff/D18613005)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qrnn_test.py:
```diff
@@ -0,0 +1,65 @@
+
+class _RNNBenchmarkBase(op_bench.TorchBenchmarkBase):
+    def init(self, input_size, hidden_size, dtype):
+        print(input_size, hidden_size, dtype)
```
This should be available in the printed result already?

z-a-f replied:
Still debugging -- I'll ping when ready for final review. Sorry about the confusion

z-a-f commented:
This is currently blocked by #31192

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qrnn_test.py:
```diff
@@ -0,0 +1,65 @@
+
+class _RNNBenchmarkBase(op_bench.TorchBenchmarkBase):
+    def init(self, input_size, hidden_size, dtype):
+        print(input_size, hidden_size, dtype)
```
Not sure if this is required?

Commit:
Finished the dynamic qLSTM benchmark on "dynamicly quantized lstm benchmarking"


Differential Revision: [D18613005](https://our.internmc.facebook.com/intern/diff/D18613005)

z-a-f commented:
```console
$> python -m pt.qrnn_test
# ----------------------------------------
# PyTorch/Caffe2 Operator Micro-benchmarks
# ----------------------------------------
# Tag : short

# Benchmarking PyTorch: QLSTM
# Mode: Eager
# Name: QLSTM_I1_H3_NL1_BTrue_DFalse_dtypetorch.qint8
# Input: I: 1, H: 3, NL: 1, B: True, D: False, dtype: torch.qint8
Forward Execution Time (us) : 4696.103

# Benchmarking PyTorch: QLSTM
# Mode: Eager
# Name: QLSTM_I1_H3_NL1_BTrue_DTrue_dtypetorch.qint8
# Input: I: 1, H: 3, NL: 1, B: True, D: True, dtype: torch.qint8
Forward Execution Time (us) : 8983.037

# Benchmarking PyTorch: QLSTM
# Mode: Eager
# Name: QLSTM_I5_H7_NL4_BTrue_DFalse_dtypetorch.qint8
# Input: I: 5, H: 7, NL: 4, B: True, D: False, dtype: torch.qint8
Forward Execution Time (us) : 18345.849

# Benchmarking PyTorch: QLSTM
# Mode: Eager
# Name: QLSTM_I5_H7_NL4_BTrue_DTrue_dtypetorch.qint8
# Input: I: 5, H: 7, NL: 4, B: True, D: True, dtype: torch.qint8
Forward Execution Time (us) : 37213.061
```

Commit:
Update on "dynamicly quantized lstm benchmarking"


Differential Revision: [D18613005](https://our.internmc.facebook.com/intern/diff/D18613005)

kostmo commented:
## CircleCI build failures summary
As of commit e9c6c96:
* **2/2** broken upstream at merge base 643ca5d (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=643ca5def2ea140f16bb064740b8de331ee465d8&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
    * You may want to rebase on the `viable/strict` branch (see its [recency history](https://dr.pytorch.org/master-viable-commits.html)):
        * If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
          ```
          git fetch viable/strict
          git rebase --onto viable/strict $(git merge-base origin/master HEAD)
          ```
        * If your commit is older than `viable/strict`:
          ```
          git fetch viable/strict
          git rebase viable/strict
          ```
* **0/2** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=e9c6c96e1167e169bf210bd1649280d8362ea056).

### 2 upstream failures recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [caffe2_onnx_py2_gcc5_ubuntu16_04_test](https://dr.pytorch.org/build-details.html?build_id=75496367)
* [caffe2_onnx_py3_6_clang7_ubuntu16_04_test](https://dr.pytorch.org/build-details.html?build_id=75496798)



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>
